conf/provisioning/dashboards/nuodb/disk.json

```diff
@@ -2352,7 +2687,7 @@
           "measurement": "JournalFsyncTime",
           "orderByTime": "ASC",
           "policy": "default",
-          "query": "SELECT mean(B) FROM (SELECT last(A) AS B FROM (SELECT mean(\"value\")  AS A FROM \"JournalFsyncTime\" WHERE \"nodetype\" = 'Storage' AND $timeFilter GROUP BY time(1m), \"host\" fill(none)) GROUP BY \"host\")",
+          "query": "SELECT mean(\"value\")  AS A FROM \"JournalFsyncTime\" WHERE \"nodetype\" = 'Storage' AND time >= ${__to}ms-1m AND time <= ${__to}ms",
```

Comment:
Should we apply the "$host" and "$db" filters as we did for the archive panels in the same dashboard?
This comment applies to all other panels in this group.

Maybe for future enhancement --- Can we somehow define the dashboard filters as template variable and then use it in the queries for different measurements across the panels?

Reply:
As for your future enhancement.  I had added something like that to a recent dashboard change.  Not sure which one.  It's not straight forward.

Reply:
what we could do is have a hidden variable set (startids) based upon the values selected for host and database.   I just encountered some issue when doing as hidden variable default was not correct and hard to figure out what was going on.   Also hidden variable is not truly hidden as it is passed through on url when using the links.